src/main/java/mil/dds/anet/resources/PersonResource.java

```diff
@@ -1,10 +1,6 @@
 package mil.dds.anet.resources;
 
-import io.leangen.graphql.annotations.GraphQLArgument;
-import io.leangen.graphql.annotations.GraphQLEnvironment;
-import io.leangen.graphql.annotations.GraphQLMutation;
-import io.leangen.graphql.annotations.GraphQLQuery;
-import io.leangen.graphql.annotations.GraphQLRootContext;
+import io.leangen.graphql.annotations.*;
```

Comment:
No wildcard imports please.

Reply:
It was due to the before commit without importing google code style. That has fixed and will not happen again.